code/src/components/AddTodo.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react"
+import { useDispatch } from "react-redux"
+import styled from "styled-components"
+import uniqid from "uniqid"
+
+import todos from "reducers/todos"
+
+const Label = styled.label`
+  margin: 10px;
+`
+
+const AddTodos = () => {
```

Comment:
It would be more consistent if both file and component were called the same: your file is named AddTodo and the component is AddTodos.